bin/gen-exercise.sh

```diff
@@ -99,8 +99,8 @@ EOL
     # Make the example file a simple copy of the solution file
     cp ${solution_file} ${example_file}
 
-    echo "Formatting new Odin files:"
-    bin/odinfmt -w ${exercises_path}
+    # echo "Formatting new Odin files:"
+    # bin/odinfmt -w ${exercises_path}
```

Comment:
Feel free to remove these, since the generated exercises should be properly formatted. 😄

Reply:
I suggest we temporarily don't use the current `bin/gen-exercise.sh`, and rather use `configlet create` to create the new exercise metadata, configuration, and blank stubs. Then let's rather spend the time on an Odin-based utility than can properly parse the JSON and create the test stubs.